### DISCUSSION TITLE
How can I find Safari datas/informations?

### DISCUSSION THREAD

**Discussion by STDoge** [MaxiHuHe04/iTunes-Backup-Explorer#34] [General]
Which domain contained the data of Safari? Like history, bookmarks, reading lists or even open tabs. Thank in advance!

**Comment by MaxiHuHe04** [comment]
Hi, in the `HomeDomain`, there is a folder called Safari. It should contain SQLite database files with the data you need. One thing to note however, on my backup at least, I found that the state is not really up to date. I don't know why that is, maybe iOS stores it somewhere else first.
